src/dymaptic.GeoBlazor.Core/Scripts/authenticationManager.ts

```diff
@@ -23,6 +23,10 @@ export default class AuthenticationManager {
             esriConfig.apiKey = apiKey;
         }
         
+        if (trustedServers !== null) {
+            esriConfig.request.trustedServers = esriConfig.request.trustedServers !== undefined ? esriConfig.request.trustedServers.concat(trustedServers) : trustedServers;
```

Comment:
would `esriConfig.request.trustedServers` ever be `undefined`? Looking at the documentation it appears it should be of type `string[]`, so even if there are no trusted servers listed and the array is empty, simply using `.concat` should work regardless without the need for the ternary operator. Please disregard if you've found otherwise.

Reply:
I wouldn't be completely surprised if @AbdelKarimCI had found the docs were wrong :) I definitely do a lot of this type of defensive coding when interfacing with ArcGIS or any other 3rd party, but I agree that it may be unnecessary.